api/src/main/java/org/openmrs/api/ObsService.java

```diff
@@ -218,7 +218,26 @@
 	 * @param obs
 	 * @throws APIException
 	 * @see #purgeObs(Obs, boolean)
-	 * @should delete the given obs from the database
+	 * @see #purgeObs(Obs, boolean)
+	 * @see #purgeObsDao(Obs, boolean)
+	 * @should delete the given obs from the database and ignore its complex data when applicable
+	 */
+	@Authorized(PrivilegeConstants.DELETE_OBS)
+	public void purgeObsDao(Obs obs) throws APIException;
```

Comment:
Ok ;-)

I'll rather provide another boolean argument to the `purgeObs(..)` then, it'll make everything simpler. I'll do that tomorrow.